assets/content/pages/work-with-us/carouselContent.json

```diff
@@ -0,0 +1,204 @@
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-9.jpg",
+          "alt": "Photo by Etienne Girardet on Unsplash"
+        }
+      },
+      {
+        "title": "State of the DAOS Newsletter Feature",
+        "paras": [
+          "State of the DAOs, the high-signal low-noise newsletter to understanding DAOs."
+        ],
+        "cta": {
+          "text": "State of the DAOs",
+          "link": "https://banklessdao.substack.com/s/state-of-the-daos?utm_source=substack&utm_medium=menu"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-10.jpg",
+          "alt": "Photo by Duy Pham on Unsplash"
+        }
+      },
+      {
+        "title": "Custom Designs",
+        "paras": ["CHANGE ME"],
+        "cta": {
+          "text": "the Design Guild",
+          "link": "https://www.notion.so/Design-Guild-6f59223e601a4e418151b3622aa6b81c"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-11.jpg",
+          "alt": "Photo by Hal Gatewood on Unsplash"
+        }
+      }
+    ]
+  },
+  {
+    "title": "Sponsorships and Marketing",
+    "description": "Scale your presence in Web3",
+    "slides": [
+      {
+        "title": "Newsletter Sponsorships",
+        "paras": [
+          "Sponsor any of our newsletters and reach tens of thousands of readers every week.",
+          "Choose from Weekly Rollup, State of the DAOs, Decentralized Art, Decentralized Law, and Gearing up, as well as a variety of languages, to reach your perfect audience. Inquire about sponsoring a newsletter by filling out this form."
+        ],
+        "cta": {
+          "text": "Bankless Newsletters",
+          "link": "https://www.notion.so/Writers-Guild-585a90523b9f4630bc04892931a11f7d"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-5.jpg",
+          "alt": "Photo by Jess Bailey on Unsplash"
+        }
+      },
+      {
+        "title": "Podcast Sponsorships",
+        "paras": ["CHANGE ME"],
+        "cta": {
+          "text": "Bankless AV Guild",
+          "link": "https://www.notion.so/bankless/AudioVisual-Guild-64f9d5b6e62a45dc93391e4d075b40a9"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-4.jpg",
+          "alt": "Photo by Matthew Kwong on Unsplash"
+        }
+      },
+      {
+        "title": "Marketing Services",
+        "paras": [
+          "Bankless Consulting is the world's first web3-native consulting firm. Whether you're a DAO or a traditional enterprise, we'll help you capture the potential of the rapidly expanding decentralized economy."
+        ],
+        "cta": {
+          "text": "Bankless Consulting",
+          "link": "https://www.banklessconsulting.com/"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-12.jpg",
+          "alt": "Photo by Jason Goodman on Unsplash"
```

Comment:
credit to the photographer, well done!

Reply:
Not fully happy with this but yeah worth attributing where possible.

I'm thinking of changing it to a <figure> where the <figcaption> is the attribution

That way we can rewrite the alt as an actual alt :shrug: